x-pack/test/plugin_api_integration/test_suites/event_log/public_api_integration.ts

```diff
@@ -156,6 +156,29 @@ export default function ({ getService }: FtrProviderContext) {
           expect(eventsUntil.length).to.be(expectedEvents.length + 1);
           assertEventsFromApiMatchCreatedEvents(eventsUntil, [firstEvent, ...expectedEvents]);
         });
+
+        it('should aggregate events summary by Saved Object ids', async () => {
```

Comment:
I feel that some tests are missing here:

1. What happens when a user requests SOs they don't have permission to see?
2. What happens when a user requests a mix of SOs they are allowed to see and SOs they're not allowed to see?
3. What happens when a user requests SOs that don't exist?

Reply:
Testing this locally, I'm not sure the behaviour I'm seeing aligns with what I'd expect.
If a user specifies a SO that doesn't exist or they have no permission to view - this just gets silently ignored.

I _think_ we should actually return errors in these cases... 404s... or 302s.... accordingly.

What do you think?

Reply:
I think 404 should be the right error message.